src/core/components/operations.jsx

```diff
@@ -33,7 +35,17 @@ export default class Operations extends React.Component {
     const Collapse = getComponent("Collapse")
 
     let showSummary = layoutSelectors.showSummary()
-    let { docExpansion, displayOperationId, displayRequestDuration } = getConfigs()
+    let { docExpansion, displayOperationId, displayRequestDuration, maxDisplayedTags } = getConfigs()
+
+    if (filter) {
+      taggedOps = taggedOps.filter((tagObj, tag) => {
+        return tag.startsWith(filter === true ? "" : filter)
```

Comment:
Why limit to startsWith ?
I think it would be better to use indexOf to find any matches.